src/apps/loyalty/server/index.tsx

```diff
@@ -71,4 +72,18 @@ app.get("/inquiries", (req, res) => {
   }))
 })
 
+app.get("/thankyou", (req, res) => {
+  if (!req.user) {
+    return res.redirect(req.baseUrl + "/login")
+  }
+
+  const html = renderToString(<Thankyou />)
+  const styles = styleSheet.rules().map(rule => rule.cssText).join("\n")
+  res.send(renderPage({
+    styles,
+    html,
+    entrypoint: "/bundles/thankyou.js",
+  }))
```

Comment:
It would be interesting to see if we can easily decide to not make certain routes isomorphic. i.e. in this case there’s no real need, I think, to load React etc on the client-side, or am I overlooking something?